### ISSUE TITLE
OpenSSH server install failing on latest Azure Windows Server 2022 images

### ISSUE THREAD

**Issue by aravindhp** [microsoft/Windows-Containers#285]
**Describe the bug**
We are seeing  the following failure when trying to install OpenSSH server on the latest Windows Server 2022 images:
```console
PS C:\ProgramData\ssh> Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
Add-WindowsCapability : Add-WindowsCapability failed. Error code = 0x800f0950
At line:1 char:1
+ Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Add-WindowsCapability], COMException
    + FullyQualifiedErrorId : Microsoft.Dism.Commands.AddWindowsCapabilityCommand
```
Image used:
```json
    "imageReference": {
      "communityGalleryImageId": null,
      "exactVersion": "20348.1131.221014",
      "id": null,
      "offer": "WindowsServer",
      "publisher": "MicrosoftWindowsServer",
      "sharedGalleryImageId": null,
      "sku": "2022-datacenter",
      "version": "latest"
    }
```
However if we pin the version to `20348.1129.221007`, the issue does now show up and we are able to install OpenSSH successfully. 

```json
    "imageReference": {
      "communityGalleryImageId": null,
      "exactVersion": "20348.1129.221007",
      "id": null,
      "offer": "WindowsServer",
      "publisher": "MicrosoftWindowsServer",
      "sharedGalleryImageId": null,
      "sku": "2022-datacenter",
      "version": "20348.1129.221007"
    }
```

**To Reproduce** 
1. Create a Windows VM specifying the latest Windows Server 2022 (I suspect all variants but have not confirmed each variant) image
2. RDP into the VM and try to install OpenSSH server using `Add-WindowsCapability -Online -Name OpenSSH.Server~~~~0.0.1.0`
3. You will see the above failure

**Expected behavior**
OpenSSH server installs successfully

**Comment by peterlymo** [comment]
no fix yet?

**Comment by vrapolinario** [comment]
Just to clarify: The behavior you described is on the Azure VM itself, right? Not on Windows containers (I honestly don't know if this is even available).
This repo is for issues on Windows containers. If this is an issue on the Azure VM, please open a ticket support with Azure here: https://learn.microsoft.com/en-us/azure/azure-portal/supportability/how-to-create-azure-support-request

**Comment by aravindhp** [comment]
Yes the issue is on the VM itself. I wanted to open an issue here for more visibility. I did open a ticket with Azure support. I did some more debugging and grabbing the latest update resolved the issue however the Azure images with bug is still present.

**Comment by fady-azmy-msft** [comment]
Thank you for highlighting this issue @aravindhp. We've reproduced the bug on our side as well, and will publish the fix soon. I'll share with you a timeline as soon as I get clarity on this.

**Comment by fady-azmy-msft** [comment]
This looks similar to another issue on (https://github.com/actions/runner-images/issues/6431), a temporary workaround would be to downgrade "image_version": "20348.1129.221007" until there's a newer version with this fixed. @akarshm any ideas on when this should be out?

**Comment by fady-azmy-msft** [comment]
Closing this issue as the this has been fixed with version `20348.1249.221105`. See this [PR for more details](https://github.com/actions/runner-images/pull/6545).
